.changeset/great-eggs-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-frontend/application-components': patch
+'@commercetools-local/visual-testing-app': patch
+---
+
+Added warnings to the `page-content-wide' and tests update
```

Comment:
Since the changeset is targeted for `app-kit` users, perhaps we can explain a little further what we're changing here.

Here's an idea:
```
We included a new warning in the `PageContentWide` component to let consumers know the component will only render its first two children when configured to use two columns but consumers provided more than two children.
```

What do you think? 🤔

Reply:
Sure, make sense. :) Updated here - [c7557ec](https://github.com/commercetools/merchant-center-application-kit/pull/3127/commits/c7557ecb5477db0f99acf7e076887da7249ff3e8)